R/palettes.R

```diff
@@ -122,3 +122,48 @@ show_ncaa_col <- function(...) {
           axis.ticks.x = element_blank())
 }
 
+#' Displays palettes for all teams for a specified sport
+#' @import ggplot2
+#' @importFrom dplyr filter
+#' @param sport character vector (basketball, soccer, football, hockey)
+#' @param ... arguments passed to other functions
+#' @export
+#' @seealso \code{\link[scales]{show_col}}
+#' @examples 
+#' show_sport_col(sport = "soccer")
+show_sport_col <- function(sport, ...){
+ sport <- tolower(sport)
+  
+  if(sport == "basketball"){
```

Comment:
Please replace the chained `if-else()` statements with a single `case_when()` statement